lib/questions.rb

```diff
@@ -223,4 +281,14 @@ def fizzbuzz_without_modulo
 # at the end.
 # (there's no RSpec test for this one)
 def ninety_nine_bottles_of_beer
+  99.downto(1) do |n|
+    nouns = Hash.new('bottles')
+    nouns[1] = 'bottle'
+
+    puts "#{n} #{nouns[n]} of beer on the wall"
+    puts "#{n} #{nouns[n]} of beer"
+    puts 'take one down, pass it around'
+    puts "#{n-1} #{nouns[n-1]} of beer on the wall\n\n" if n > 1
```

Comment:
Surrounding space missing for operator '-'.<br>Do not write to stdout. Use Rails' logger if you want to log.